Packs/RSANetWitness_v11_1/Integrations/RSANetWitnessv115/RSANetWitnessv115.yml

```diff
@@ -1991,13 +2002,12 @@ script:
       required: true
     description: Restores the network connection and removes IP addresses added to the exclusion list for the host with the specified agent ID.
     name: rsa-nw-endpoint-isolation-remove
-  dockerimage: demisto/python3:3.10.12.63474
+  dockerimage: demisto/python3:3.10.12.65389
   isfetch: true
-  runonce: false
   script: '-'
   subtype: python3
   type: python
 tests:
 - RSANetWitnessv115-Test
-defaultmapperin: RSA NetWitness v11.5 - incoming mapper
+- defaultmapperin: RSA NetWitness v11.5 - incoming mapper
```

Comment:
why does git show an added `-`? 🤔 
If by testing manually the behavior is the same as before, resolve this issue.

Reply:
done